packages/orderbook/test/js_api/transaction.test.ts

```
@@ -20,11 +20,8 @@ describe('Rain Orderbook JS API Package Bindgen Tests - SgTransaction', async fu
 			.forPost('/sg1')
 			.thenReply(200, JSON.stringify({ data: { transaction: transaction1 } }));
 
-		try {
-			const result: SgTransaction = await getTransaction(mockServer.url + '/sg1', transaction1.id);
-			assert.equal(result.id, transaction1.id);
-		} catch (e) {
-			assert.fail('expected to resolve, but failed' + (e instanceof Error ? e.message : String(e)));
-		}
+		const result = await getTransaction(mockServer.url + '/sg1', transaction1.id);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)